plugins/signals-node/src/types.ts

```diff
@@ -25,7 +25,9 @@ export type SignalsServiceOptions = {
 
 /** @public */
 export type SignalPayload<TMessage extends JsonObject = JsonObject> = {
-  recipients: string[] | string | null;
+  recipients:
+    | { type: 'user'; entityRefs: string | string[] }
```

Comment:
For notifications this is `type: 'entity', entityRef: string | string[]`. Thinking with might as well mirror that?

Reply:
Yeah, it is but there's resolving of the notification receivers in the notifications backend from other than user entity references. We don't really want to duplicate that logic to signals as it would require also catalog API. Signals are either broadcast or directly to users

Reply:
There's the minor thing of singular vs plural `ref(s)` too I guess